mainari.py

```diff
@@ -0,0 +1,123 @@
+import requests
+import json
+import threading
+
+import logging as log
+
+class Mainari:
+    def __init__(self, server, game_ops='', server_admins=''):
+        self.api_url = 'https://api.mcsrvstat.us/1/'
+        self.commands = { 'minecraft': self.getServerInfo }
+        self.is_in_cooldown = False
+        self.game_ops = self.parseNicks(game_ops)
+        self.server = server
+        self.server_admins = self.parseNicks(server_admins)
+
+
+    def getCommands(self):
+        return self.commands
+
+
+    def getServerInfo(self, bot, update, args=''):
+        if self.is_in_cooldown:
+            bot.sendMessage(chat_id=update.message.chat_id, text="Cool. Cool cool cool.")
+            return
+        
+        # Set the cooldown to prevent API abuse and blocking of the bot
+        threading.Timer(45.0, self.resetInfoCooldown).start()
+        self.is_in_cooldown = True
+
+        r = requests.get(self.api_url+ self.server)
+        data = r.json()
+
+        message_text = self.parseServerData(data)
+        bot.sendMessage(chat_id=update.message.chat_id, parse_mode='Markdown', text=message_text)
+
+    
+    def parseNicks(self, nicks):
+        nick_list = nicks.split(',')
+        nick_list = [nick.strip() for nick in nick_list]
+        return nick_list
+
+
+    def parseServerData(self, data):
+        # Data gathered if the server is offline
+        try:
+            server_is_offline = data['offline']
```

Comment:
You can check if key is in dict with `if 'offline' in data:` instead of try except.